charts/ratify/values.yaml

```diff
@@ -82,7 +82,10 @@ provider:
     type: ristretto # cache type, currently only ristretto(default) and redis are supported
     cacheSizeMb: 256 # max size of the cache in MB
     ttl: 10s # cache ttl duration
+    size: 100 # number of items to be kept in cache. Set to 0 to disable cache.
```

Comment:
is the cache size change related to mutation?

Reply:
Hi @susanshi , It was not related. It showed as diff because I merged main branch to remove conflicts . I have re-based my branch and now it shows actual changes.